.github/workflows/static_build.yml

```diff
@@ -13,58 +13,77 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  micromamba_full_static:
+  conda-build:
+    name: "micromamba - ${{ matrix.platform }}-${{ matrix.arch }}"
     runs-on: ${{ matrix.os }}
+    env:
+      CI: True
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        include:
+          - { os: ubuntu-latest, platform: linux, arch: 64 }
+          - { os: ubuntu-latest, platform: linux, arch: aarch64 }
+          - { os: ubuntu-latest, platform: linux, arch: ppc64le }
+          - { os: macos-latest, platform: osx, arch: 64 }
+          - { os: macos-latest, platform: osx, arch: arm64 }
     steps:
-      - uses: actions/checkout@v3
-      - name: create build environment
-        uses: mamba-org/provision-with-micromamba@main
+      - name: Checkout branch
+        uses: actions/checkout@v3
+      - name: Checkout micromamba-feedstock
+        uses: actions/checkout@v3
         with:
-          environment-file: ./micromamba/environment-dev.yml
-          environment-name: build_env
-          cache-env: true
-      - name: create build environment
-        run: micromamba install -y -n build_env -f ./libmamba/environment-static-dev.yml
-      - uses: hendrikmuhs/ccache-action@main
+          repository: conda-forge/micromamba-feedstock
+          path: micromamba-feedstock
+      - name: Patch micromamba-feedstock
+        if: ${{ matrix.platform == 'linux' }}
+        run: |
+          sed -i -E ':a; N; $!ba; s/url:.*libmamba.patch/git_url: ..\/..\//g' micromamba-feedstock/recipe/meta.yaml
```

Comment:
What is this magic sed command btw? We should document it or find a simpler more obvious alternative.

Reply:
This replaces the
``` 
url: ...
sha256: ...
patches: 
  libmamba.patch
```

with a simple `git_url: ../../`

Reply:
The problem is that sed is not working that well for multiple lines... On macOS, this command doesn't work and I didn't manage to find a single sed command so I split it in four sed commands... Do you know a more elegant alternative for replacement of multiple lines by a single one?

Reply:
Unfortunately I don't :(

Specifically I was wondering what this means: `:a; N; $!ba; s/u`

Reply:
I copied it somewhere from stack overflow where there was an explanation 😅 but I can't find it anymore :/

Reply:
Hmm, I guess `sed -i -E -z 's/url:.*libmamba.patch/git_url: ..\/..\//g'` works too...